HTMLToQPDF/Components/HTMLComponent.cs

```diff
@@ -52,6 +53,14 @@ internal class HTMLComponent : IComponent
 
         public void Compose(IContainer container)
         {
+            var fontPath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "Assets\\Fonts");
```

Comment:
you want pathseperatorcharacter instead of \\?

Reply:
updated in most recent commit